### DISCUSSION TITLE
How to use iptables?

### DISCUSSION THREAD

**Discussion by DevGoran** [wg-easy/wg-easy#1423] [Q&A]
Hi

```
module.exports.WG_POST_UP = process.env.WG_POST_UP || `
iptables -t nat -A POSTROUTING -s ${module.exports.WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o ${module.exports.WG_DEVICE} -j MASQUERADE;
iptables -A INPUT -p udp -m udp --dport ${module.exports.WG_PORT} -j ACCEPT;
iptables -A FORWARD -i wg0 -j ACCEPT;
iptables -A FORWARD -o wg0 -j ACCEPT;
`.split('\n').join(' ');
```

So, from my understanding as soon as I add something to the environment variable, it will replace the default settings, which means if I want to have the same rule set and one additional for a specific client (more restricted), I need to add the default iptables rules as well in the environment variable?

If so, how should the environment variable in my compose file look exactly? Seems like whatever I try traffic does not get routed through to my local network for any device.

What I want to achieve is following:
- Default behaviour wg-easy has
- Additionally, for one single peer I want to have some restrictions: Don't allow 10.8.0.20 to access any resources on the LAN of 10.8.0.1 and allow only SSH and HTTPS to be access on the LAN of 10.8.0.20

How can I achieve this?

**Comment by kaaax0815** [comment]
You can look at `wg0.conf` and the PostUp field. Copy this and add your own rules and set this as the `WG_POST_UP` env var
